[2024-10-07T23:22:31.123-0300] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-10-07T23:22:31.160-0300] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: preprocess_predict.preprocess manual__2024-10-08T02:22:29.590970+00:00 [queued]>
[2024-10-07T23:22:31.178-0300] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: preprocess_predict.preprocess manual__2024-10-08T02:22:29.590970+00:00 [queued]>
[2024-10-07T23:22:31.180-0300] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2024-10-07T23:22:31.202-0300] {taskinstance.py:2330} INFO - Executing <Task(_PythonVirtualenvDecoratedOperator): preprocess> on 2024-10-08 02:22:29.590970+00:00
[2024-10-07T23:22:31.217-0300] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=3932) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-10-07T23:22:31.220-0300] {standard_task_runner.py:63} INFO - Started process 3933 to run task
[2024-10-07T23:22:31.219-0300] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'preprocess_predict', 'preprocess', 'manual__2024-10-08T02:22:29.590970+00:00', '--job-id', '87', '--raw', '--subdir', 'DAGS_FOLDER/Predict/preprocess_predict.py', '--cfg-path', '/tmp/tmp9z0enml5']
[2024-10-07T23:22:31.222-0300] {standard_task_runner.py:91} INFO - Job 87: Subtask preprocess
[2024-10-07T23:22:31.294-0300] {task_command.py:426} INFO - Running <TaskInstance: preprocess_predict.preprocess manual__2024-10-08T02:22:29.590970+00:00 [running]> on host 4a07270c49a4
[2024-10-07T23:22:31.414-0300] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='preprocess_predict' AIRFLOW_CTX_TASK_ID='preprocess' AIRFLOW_CTX_EXECUTION_DATE='2024-10-08T02:22:29.590970+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-10-08T02:22:29.590970+00:00'
[2024-10-07T23:22:31.415-0300] {taskinstance.py:430} INFO - ::endgroup::
[2024-10-07T23:22:31.418-0300] {process_utils.py:183} INFO - Executing cmd: /home/***/.local/bin/python -m virtualenv /tmp/venvnufqj30p --python=python
[2024-10-07T23:22:31.420-0300] {process_utils.py:187} INFO - Output:
[2024-10-07T23:22:32.130-0300] {process_utils.py:191} INFO - created virtual environment CPython3.12.3.final.0-64 in 201ms
[2024-10-07T23:22:32.131-0300] {process_utils.py:191} INFO -   creator CPython3Posix(dest=/tmp/venvnufqj30p, clear=False, no_vcs_ignore=False, global=False)
[2024-10-07T23:22:32.132-0300] {process_utils.py:191} INFO -   seeder FromAppData(download=False, pip=bundle, via=copy, app_data_dir=/home/***/.local/share/virtualenv)
[2024-10-07T23:22:32.133-0300] {process_utils.py:191} INFO -     added seed packages: pip==24.2
[2024-10-07T23:22:32.135-0300] {process_utils.py:191} INFO -   activators BashActivator,CShellActivator,FishActivator,NushellActivator,PowerShellActivator,PythonActivator
[2024-10-07T23:22:32.166-0300] {process_utils.py:183} INFO - Executing cmd: /tmp/venvnufqj30p/bin/pip install -r /tmp/venvnufqj30p/requirements.txt
[2024-10-07T23:22:32.167-0300] {process_utils.py:187} INFO - Output:
[2024-10-07T23:22:34.017-0300] {process_utils.py:191} INFO - Collecting joblib (from -r /tmp/venvnufqj30p/requirements.txt (line 1))
[2024-10-07T23:22:34.020-0300] {process_utils.py:191} INFO -   Using cached joblib-1.4.2-py3-none-any.whl.metadata (5.4 kB)
[2024-10-07T23:22:34.262-0300] {process_utils.py:191} INFO - Collecting numpy (from -r /tmp/venvnufqj30p/requirements.txt (line 2))
[2024-10-07T23:22:34.265-0300] {process_utils.py:191} INFO -   Using cached numpy-2.1.2-cp312-cp312-manylinux_2_17_x86_64.manylinux2014_x86_64.whl.metadata (60 kB)
[2024-10-07T23:22:34.492-0300] {process_utils.py:191} INFO - Collecting pandas (from -r /tmp/venvnufqj30p/requirements.txt (line 3))
[2024-10-07T23:22:34.494-0300] {process_utils.py:191} INFO -   Using cached pandas-2.2.3-cp312-cp312-manylinux_2_17_x86_64.manylinux2014_x86_64.whl.metadata (89 kB)
[2024-10-07T23:22:34.696-0300] {process_utils.py:191} INFO - Collecting scikit-learn (from -r /tmp/venvnufqj30p/requirements.txt (line 4))
[2024-10-07T23:22:34.698-0300] {process_utils.py:191} INFO -   Using cached scikit_learn-1.5.2-cp312-cp312-manylinux_2_17_x86_64.manylinux2014_x86_64.whl.metadata (13 kB)
[2024-10-07T23:22:34.769-0300] {process_utils.py:191} INFO - Collecting python-dateutil>=2.8.2 (from pandas->-r /tmp/venvnufqj30p/requirements.txt (line 3))
[2024-10-07T23:22:34.772-0300] {process_utils.py:191} INFO -   Using cached python_dateutil-2.9.0.post0-py2.py3-none-any.whl.metadata (8.4 kB)
[2024-10-07T23:22:34.874-0300] {process_utils.py:191} INFO - Collecting pytz>=2020.1 (from pandas->-r /tmp/venvnufqj30p/requirements.txt (line 3))
[2024-10-07T23:22:34.876-0300] {process_utils.py:191} INFO -   Using cached pytz-2024.2-py2.py3-none-any.whl.metadata (22 kB)
[2024-10-07T23:22:34.918-0300] {process_utils.py:191} INFO - Collecting tzdata>=2022.7 (from pandas->-r /tmp/venvnufqj30p/requirements.txt (line 3))
[2024-10-07T23:22:34.921-0300] {process_utils.py:191} INFO -   Using cached tzdata-2024.2-py2.py3-none-any.whl.metadata (1.4 kB)
[2024-10-07T23:22:35.137-0300] {process_utils.py:191} INFO - Collecting scipy>=1.6.0 (from scikit-learn->-r /tmp/venvnufqj30p/requirements.txt (line 4))
[2024-10-07T23:22:35.140-0300] {process_utils.py:191} INFO -   Using cached scipy-1.14.1-cp312-cp312-manylinux_2_17_x86_64.manylinux2014_x86_64.whl.metadata (60 kB)
[2024-10-07T23:22:35.359-0300] {process_utils.py:191} INFO - Collecting threadpoolctl>=3.1.0 (from scikit-learn->-r /tmp/venvnufqj30p/requirements.txt (line 4))
[2024-10-07T23:22:35.362-0300] {process_utils.py:191} INFO -   Using cached threadpoolctl-3.5.0-py3-none-any.whl.metadata (13 kB)
[2024-10-07T23:22:35.395-0300] {process_utils.py:191} INFO - Collecting six>=1.5 (from python-dateutil>=2.8.2->pandas->-r /tmp/venvnufqj30p/requirements.txt (line 3))
[2024-10-07T23:22:35.397-0300] {process_utils.py:191} INFO -   Using cached six-1.16.0-py2.py3-none-any.whl.metadata (1.8 kB)
[2024-10-07T23:22:35.425-0300] {process_utils.py:191} INFO - Using cached joblib-1.4.2-py3-none-any.whl (301 kB)
[2024-10-07T23:22:35.428-0300] {process_utils.py:191} INFO - Using cached numpy-2.1.2-cp312-cp312-manylinux_2_17_x86_64.manylinux2014_x86_64.whl (16.0 MB)
[2024-10-07T23:22:35.448-0300] {process_utils.py:191} INFO - Using cached pandas-2.2.3-cp312-cp312-manylinux_2_17_x86_64.manylinux2014_x86_64.whl (12.7 MB)
[2024-10-07T23:22:35.466-0300] {process_utils.py:191} INFO - Using cached scikit_learn-1.5.2-cp312-cp312-manylinux_2_17_x86_64.manylinux2014_x86_64.whl (12.9 MB)
[2024-10-07T23:22:35.482-0300] {process_utils.py:191} INFO - Using cached python_dateutil-2.9.0.post0-py2.py3-none-any.whl (229 kB)
[2024-10-07T23:22:35.485-0300] {process_utils.py:191} INFO - Using cached pytz-2024.2-py2.py3-none-any.whl (508 kB)
[2024-10-07T23:22:35.489-0300] {process_utils.py:191} INFO - Using cached scipy-1.14.1-cp312-cp312-manylinux_2_17_x86_64.manylinux2014_x86_64.whl (40.8 MB)
[2024-10-07T23:22:35.545-0300] {process_utils.py:191} INFO - Using cached threadpoolctl-3.5.0-py3-none-any.whl (18 kB)
[2024-10-07T23:22:35.547-0300] {process_utils.py:191} INFO - Using cached tzdata-2024.2-py2.py3-none-any.whl (346 kB)
[2024-10-07T23:22:35.549-0300] {process_utils.py:191} INFO - Using cached six-1.16.0-py2.py3-none-any.whl (11 kB)
[2024-10-07T23:22:35.820-0300] {process_utils.py:191} INFO - Installing collected packages: pytz, tzdata, threadpoolctl, six, numpy, joblib, scipy, python-dateutil, scikit-learn, pandas
[2024-10-07T23:22:51.547-0300] {process_utils.py:191} INFO - Successfully installed joblib-1.4.2 numpy-2.1.2 pandas-2.2.3 python-dateutil-2.9.0.post0 pytz-2024.2 scikit-learn-1.5.2 scipy-1.14.1 six-1.16.0 threadpoolctl-3.5.0 tzdata-2024.2
[2024-10-07T23:22:51.982-0300] {process_utils.py:183} INFO - Executing cmd: /tmp/venvnufqj30p/bin/python /tmp/venv-callh6rp7fuz/script.py /tmp/venv-callh6rp7fuz/script.in /tmp/venv-callh6rp7fuz/script.out /tmp/venv-callh6rp7fuz/string_args.txt /tmp/venv-callh6rp7fuz/termination.log
[2024-10-07T23:22:51.983-0300] {process_utils.py:187} INFO - Output:
[2024-10-07T23:22:52.070-0300] {process_utils.py:191} INFO - Traceback (most recent call last):
[2024-10-07T23:22:52.071-0300] {process_utils.py:191} INFO -   File "/tmp/venv-callh6rp7fuz/script.py", line 63, in <module>
[2024-10-07T23:22:52.073-0300] {process_utils.py:191} INFO -     res = task_preprocess(*arg_dict["args"], **arg_dict["kwargs"])
[2024-10-07T23:22:52.074-0300] {process_utils.py:191} INFO -           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
[2024-10-07T23:22:52.075-0300] {process_utils.py:191} INFO - TypeError: task_preprocess() missing 1 required positional argument: 'ti'
[2024-10-07T23:22:52.363-0300] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-10-07T23:22:52.380-0300] {taskinstance.py:2905} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 465, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 432, in _execute_callable
    return execute_callable(context=context, **execute_callable_kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 400, in wrapper
    return func(self, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/decorators/base.py", line 265, in execute
    return_value = super().execute(context)
                   ^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 400, in wrapper
    return func(self, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/operators/python.py", line 437, in execute
    return super().execute(context=serializable_context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 400, in wrapper
    return func(self, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/operators/python.py", line 235, in execute
    return_value = self.execute_callable()
                   ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/operators/python.py", line 759, in execute_callable
    result = self._execute_python_callable_in_subprocess(python_path)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/operators/python.py", line 514, in _execute_python_callable_in_subprocess
    raise AirflowException(error_msg) from None
airflow.exceptions.AirflowException: Process returned non-zero exit status 1.
task_preprocess() missing 1 required positional argument: 'ti'
[2024-10-07T23:22:52.387-0300] {taskinstance.py:1206} INFO - Marking task as FAILED. dag_id=preprocess_predict, task_id=preprocess, run_id=manual__2024-10-08T02:22:29.590970+00:00, execution_date=20241008T022229, start_date=20241008T022231, end_date=20241008T022252
[2024-10-07T23:22:52.413-0300] {standard_task_runner.py:110} ERROR - Failed to execute job 87 for task preprocess (Process returned non-zero exit status 1.
task_preprocess() missing 1 required positional argument: 'ti'; 3933)
[2024-10-07T23:22:52.439-0300] {local_task_job_runner.py:240} INFO - Task exited with return code 1
[2024-10-07T23:22:52.469-0300] {taskinstance.py:3498} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-10-07T23:22:52.476-0300] {local_task_job_runner.py:222} INFO - ::endgroup::
